addons/purchase_edi_ubl_bis3/__manifest__.py

```diff
@@ -1,6 +1,5 @@
 {
     'name': "Import/Export electronic orders with UBL",
-    'version': '1.0',
```

Comment:
Reason for removing the version of the module?

Reply:
Probably because I asked for the removal in the sale module ^^.
Specifying 1.0 is useless since it's the default manifest value, but it doesn't have to be removed from the existing code indeed.

Reply:
Yes my bad I removed it from here instead sale edi :cry: